addons/website_theme_install/models/ir_module_module.py

```diff
@@ -162,7 +162,8 @@ def _update_records(self, model_name, website):
                             rec_data.pop('active')
                         find.update(rec_data)
                 else:
-                    self.env[model_name].create(rec_data)
+                    new_rec = self.env[model_name].create(rec_data)
+                    rec._post_process_creation(new_rec)
```

Comment:
what's about update 2 line up ?

Don't we need to handle translation in this case?

Reply:
good catch, this won't overwrite existing translations but I guess it is ok (or is it?)